src/PlexRequests/Models/CreateAdminResult.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PlexRequests.Models
+{
+    public class CreateAdminResult
```

Comment:
Duplicate class from UserLoginResult. If they return the same result, might worth seeing if  you want to merge them. Not big issues though. If you dont merge them, may be use the same class?